djangocms_version_locking/check.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.contenttypes.models import ContentType
+
+from .models import VersionLock
+
+
+def content_is_not_locked(placeholder, user):
```

Comment:
The naming of this had me triple check the tests for logic. Could be better as content_is_unlocked. Personal preference which may have made it easier for me to follow the test logic.